### PR TITLE
fix(DepartureTimes): Hide tomorrow when the time is relative

### PR DESCRIPTION
**Pull request by kotva006**
#### Summary of changes
Remove `Tomorrow` from showing up on the time on the departure cards, and departure list.
This only happens if the time is set to relative.

**Asana Ticket:** [Departure Times | Text Mis Aligned close to midnight](https://app.asana.com/0/555089885850811/1205225444022104)

---

#### General checks
* [ ] **Are the changes organized into self-contained commits with descriptive and well-formatted commit messages?** This is a good practice that can facilitate easier reviews.
* [ ] **Testing.** Do the changes include relevant passing updates to tests? This includes updating screenshots. Preferably tests are run locally to verify that there are no test failures created by these changes, before opening a PR.
* [ ] **Tech debt.** Have you checked for tech debt you can address in the area you're working in? This can be a good time to address small issues, or create Asana tickets for larger issues.

#### New UI, or substantial UI changes
* [ ] **Cross-browser compatibility** is less of an issue now that we're no longer supporting IE, but changes still need to work as expected in Safari, Chrome, and Firefox.
* [ ] **Are interactive elements accessible?** This includes at minimum having [relevant keyboard interactions](https://thinkdobecreate.com/articles/4-required-tests-before-shipping-new-features/#keyboard-interaction) and [visible focus](https://thinkdobecreate.com/articles/4-required-tests-before-shipping-new-features/#visible-focus), but can also include verification with screen reader testing.
* [ ] **Other accessibility checks** such as sufficient [color constrast](https://thinkdobecreate.com/articles/4-required-tests-before-shipping-new-features/#color-contrast), or whether the layout holds up at 200% [zoom level](https://thinkdobecreate.com/articles/4-required-tests-before-shipping-new-features/#visible-focus).

#### New endpoints, or non-trivial changes to current endpoints
* [ ] **Have we load-tested** any new pages or internal API endpoints that will receive significant traffic? [See load testing docs](./../apps/site/load_tests/README.md)
* [ ] **If this change involves routes,** does it work correctly with pertinent "unusual" routes such as the combined Green Line, Silver Line, Foxboro commuter rail, and single-direction bus routes like the 170?

### CODE REVIEW

File: apps/site/assets/ts/stop/components/DisplayTime.tsx [hunk 1]
```diff
@@ -121,7 +121,10 @@ const DisplayTime = ({
             )}{" "}
             {/* Prioritize displaying Tomorrow over track name if both are present */}
             <span className="time-details">
-              {tomorrow ? "Tomorrow" : trackName || null}
+              {/* Only show tomorrow if time is not displayed in relative time */}
+              {tomorrow && !(!(isCancelled || isSkipped) && isCR)
```
Comment (i0sea) [inline apps/site/assets/ts/stop/components/DisplayTime.tsx:125]: nitpick: is there a clearer way to write this? maybe `tomorrow && !isCR && !(isCanCelled || isSkipped)`

**Review by i0sea — APPROVED** [review]

File: apps/site/assets/ts/stop/components/DisplayTime.tsx (hunk 1 above)
Comment (kotva006) [reply apps/site/assets/ts/stop/components/DisplayTime.tsx:125]: Yeah I can change it.  The idea behind this way was it is the `not` of the condition that causes the time to display as relative.

File: apps/site/assets/ts/stop/__tests__/DisplayTimeTest.tsx [hunk 2]
```diff
@@ -56,6 +55,7 @@ describe("DisplayTime", () => {
           isCR={false}
         />
       );
+      screen.debug();
```
Comment (i0sea) [inline apps/site/assets/ts/stop/__tests__/DisplayTimeTest.tsx:58]: remove debug statement

**Review by i0sea — COMMENTED** [review]
Seems to be some linting issues

**Pull request merged**